src/pages/myAdmin/guides/gettingStarted.tsx

```diff
@@ -46,14 +44,11 @@ const authenticateApi: ReactNode = (
         <code className="small-code-sample">{`{"result":{"userId":"x12345x2-172x-4d04-8xx2-xx9e088c5xxx","sessionId":"cff4e88b-931b-4363-ae4f-35b5ed169133","lastLogonDate":"2013-11-04T15:01:00.000Z","roles":[{"comments":"Third Party Integrator role","name":"Third-Party-Integrator"}],"name":"user@geotab.com"}}`}</code>
         <p>
             The API key is found in the <code className="small-code-sample">userId</code> property and the session ID is found in the <code className="small-code-sample">sessionId</code> property. Both of these properties must be included on all other API calls. If the session ID has expired, the API will return a <code className="small-code-sample">SessionExpiredException</code> which requires a call to Authenticate to get the new session ID. For more information, see the{" "}
-            {/* TODO: update link */}
-            <Link to="">Reference</Link>{" "}
+            <Link to="./../../apiReference/methods">Reference</Link>{" "}
```

Comment:
for all these, please copy the path from App.tsx

Reply:
Has been updated to "/myAdmin/apiReference/methods"